README.adoc

```diff
@@ -574,6 +574,30 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping [[skipping]]
+
+Prefer `skip` over `return` for skipping test cases.
```

Comment:
`runnable methods` may be more distinguishable from subclasses of `Minitest::Test` than `test cases`.

```suggestion
Prefer `skip` over `return` for skipping runnable methods.
```